web/ui/dashboard/src/app/private/components/create-subscription/create-subscription.component.html

```diff
@@ -184,17 +184,20 @@ <h3 class="font-semibold mb-6px">Endpoint</h3>
 	<button convoy-button size="sm" class="mt-20px" (click)="onAction.emit({ action: 'cancel' })">Close</button>
 </div>
 
-<div convoy-modal position="full" *ngIf="showFilterForm" (closeModal)="showFilterForm = false">
-	<convoy-create-subscription-filter (close)="showFilterForm = false" [schema]="subscriptionForm.get('filter_config.filter')?.value" [action]="action" (filterSchema)="getFilterSchema($event)"></convoy-create-subscription-filter>
-</div>
-
-<div convoy-modal position="center" size="sm" *ngIf="confirmModal" (closeModal)="confirmModal = false">
-	<div class="px-20px py-30px">
+<!-- subscription filters form   -->
+<dialog #filterDialog convoy-dialog position="full">
+	<convoy-create-subscription-filter (close)="filterDialog.close()" [schema]="subscriptionForm.get('filter_config.filter')?.value" [action]="action" (filterSchema)="getFilterSchema($event)"></convoy-create-subscription-filter>
+</dialog>
+
+<!-- conformation prompt  -->
+<dialog #confirmationDialog convoy-dialog position="center" size="sm">
```

Comment:
Basically, I think we should get rid of confirmation buttons in a form that's in a modal. Let me know if you think otherwise.